src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1117,6 +1118,64 @@ public void SetAvailableOrientations( List<Window.WindowOrientation> orientation
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Adds a page to the window.
+        /// </summary>
+        /// <param name="page">Page to add.</param>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AddPage(Page page)
```

Comment:
Why add only one page per type? is it by design?

Reply:
Hello, Mr. Lee, Page is the base class of ContentPage.

Reply:
I know...
In the previous review, I asked about this method implementation accepting only one instance per Type as a Dictionary.
It seems to be okay now.